src/Uno.UI/UI/Xaml/Controls/Primitives/Selector.cs

```diff
@@ -21,6 +21,8 @@ namespace Windows.UI.Xaml.Controls.Primitives
 	public partial class Selector : ItemsControl
 	{
 		private protected ScrollViewer m_tpScrollViewer;
+		private protected bool _changingSelectedIndex;
+		private protected bool _isSelectionActive;
```

Comment:
_isSelectionActive sounds more like for "selection is set" rather than "we are currently updating selection":
```suggestion
		private protected bool _isUpdatingSelection;
```